src/lib/progress-spinner/progress-spinner.spec.ts

```diff
@@ -108,6 +108,29 @@ describe('MdProgressSpinner', () => {
     expect(progressElement.componentInstance.interdeterminateInterval).toBeFalsy();
   });
 
+  it('should allow a custom stoke width', () => {
+    let fixture = TestBed.createComponent(BasicProgressSpinner);
+    let pathElement = fixture.nativeElement.querySelector('path') as HTMLElement;
+
+    fixture.detectChanges();
+
+    expect(parseInt(pathElement.style.strokeWidth))
```

Comment:
There is a need for that. Safari 9 adds the `px` suffix to the `strokeWidth` value. We could have also done it through `toContain` but `parseInt` and `toBe` is more safe here.